### ISSUE TITLE
vandium.validation 

### ISSUE THREAD

**Issue by vanerleo** [vandium-io/vandium-node#8]
How to accept all params, but force validation of only one?

each of my lambda's receive a bunch of params : body,stage, authorization ... etc
vandium.validation({
  body: vandium.types.any(),
  pathParams: vandium.types.any(),
  queryParams: vandium.types.any(),
  stage: vandium.types.any(),
  authorization: vandium.types.any(),
  isOffline: vandium.types.any(),
  country: vandium.types.string().min(1).max(50).required()
});

how do i not type all of that in every function? 
ie in this instance i'd like to validate 'country' only ...

**Comment by richardhyatt** [comment]
you can have validation ignore the other values by calling:

```js
`vandium.validation.ignore( 'body', 'pathParams', 'queryParams', 'stage', 'authorization', 'isOffline' );
```

You can call `vandium.validation.ignore()` multiple times if needed.

We thought about having a validation flag to ignore everything that wasn't declared in the validation code, but it goes against security best practices of knowing your data.

**Comment by vanerleo** [comment]
for url : /details?seoName=koelpin---mills-main-office
getting *** vandium - SQL Injection detected - EQUALS_WITH_COMMENT

thats not entirely correct.

using :

`vandium.validation({`
`  seoName: vandium.types.string().min(1).max(250).required()`
`})`

**Comment by richardhyatt** [comment]
fixed and published to vandium 2.7.3 - run: npm install vandium
